### DISCUSSION TITLE
Domain throttling and IP rotation

### DISCUSSION THREAD

**Discussion by pitbullcan** [postalserver/postal#1545] [Help with using Postal]
Please I do not know if this question has been asked before but I couldnt find it after searching.
1) is there a way to throttle the destination domain per IP
2) how can I achieve IP rotation of 100 emails per IP per hour.
Thanks.

**Comment by willpower232** [comment]
it sounds like you want to make use of IP Pools https://docs.postalserver.io/features/ip-pools

It can't limit but you should be able to achieve the randomness of IP usage

**Reply by creative-andrew** [reply]
@willpower232 can this be accomplished using IP TABLES? 

Something like this:

```
# iptables -t nat -I POSTROUTING -m state --state NEW -p tcp --dport 25 -o eth0 -m statistic --mode nth --every 5 -j SNAT --to-source 202.XXX.XX.2
# iptables -t nat -I POSTROUTING -m state --state NEW -p tcp --dport 25 -o eth0 -m statistic --mode nth --every 5 -j SNAT --to-source 202.XXX.XX.3
# iptables -t nat -I POSTROUTING -m state --state NEW -p tcp --dport 25 -o eth0 -m statistic --mode nth --every 5 -j SNAT --to-source 202.XXX.XX.4
# iptables -t nat -I POSTROUTING -m state --state NEW -p tcp --dport 25 -o eth0 -m statistic --mode nth --every 5 -j SNAT --to-source 202.XXX.XX.5
# iptables -t nat -I POSTROUTING -m state --state NEW -p tcp --dport 25 -o eth0 -m statistic --mode nth --every 5 -j SNAT --to-source 202.XXX.XX.6
done
```

**Reply by willpower232** [reply]
I don't know enough about iptables to say whether or not this will work
